### ISSUE TITLE
Corona Warn App zeigt Fehler 13 länger als 24h auf Ios

### ISSUE THREAD

**Issue by jessinger179** [corona-warn-app/cwa-app-ios#4267]
## Avoid duplicates
* [ ] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Bug is not already reported in another issue

## Technical details

- Device name:
- iOS version:
- App version:

**Comment by Ein-Tim** [comment]
Hallo @jessinger179, hast du die neuste Version der App installiert? Das ist Version 2.18.1.

**Comment by jessinger179** [comment]
Ja. Die habe ich vorgestern installiert und seitdem geht garnichts mehr.

**Comment by jessinger179** [comment]
> Hallo @jessinger179, hast du die neuste Version der App installiert? Das ist Version 2.18.1.

Ja. Die habe ich vorgestern installiert und seitdem geht garnichts mehr

**Comment by thomasaugsten** [comment]
@jessinger179 Kannst du bitte ein Screenshot bereitstellen und nochmal überprüfen das unter Appinformation wirklich v2.18 installiert ist

**Comment by jessinger179** [comment]
> @jessinger179 Kannst du bitte ein Screenshot bereitstellen und nochmal überprüfen das unter Appinformation wirklich v2.18 installiert ist

![1A77DD05-E73E-4DFD-AAF7-B5617A77CF2C](https://user-images.githubusercontent.com/100413215/155712618-47906035-9c39-4814-a5e6-3ee01570bc2c.png)
![Uploading 1759214F-015F-4928-B0F7-8DE0B3F2ADC9.png…]()

**Comment by jessinger179** [comment]
![6EC91E22-FB0C-452C-8920-E05C4E7884C3](https://user-images.githubusercontent.com/100413215/155712709-54b4bcef-c8c0-4b7b-9809-9ff244a742e2.png)

**Comment by jessinger179** [comment]
![Uploading 99AA9D35-6F11-4A47-975B-731B35EA5150.png…]()

**Comment by Ein-Tim** [comment]
@jessinger179 Wenn du ganz nach unten scrollst und auf "App-Informationen" klickst, was steht als Version ganz unten? 2.18.1?

**Comment by jessinger179** [comment]
> @jessinger179 Wenn du ganz nach unten scrollst und auf "App-Informationen" klickst, was steht als Version ganz unten? 2.18.1?

![Uploading FB65ECE7-0B3D-477B-8BFF-85A932FB6F21.png…]()

**Comment by jessinger179** [comment]
![Uploading AF074AE9-A98B-46F4-9D5D-89366A8555A4.png…]()

**Comment by Ein-Tim** [comment]
@jessinger179 Du musst warten, bis da nicht mehr Uploading... steht. Sonst können wir das Bild nicht sehen.

**Comment by jessinger179** [comment]
![Uploading 7F07D95A-5DBE-43B1-83AB-FDE0B6A9C0AC.png…]()

**Comment by jessinger179** [comment]
Das Bild wird nicht geladen, es ist aber die neueste Version geladen.

**Comment by jessinger179** [comment]
![Uploading FB0467C2-A826-4BCE-A628-6CADD5D27256.png…]()

**Comment by jessinger179** [comment]
![F183E8A2-6E61-43F8-B968-7FA6D17739C5](https://user-images.githubusercontent.com/100413215/155714972-405122e3-b141-4e92-a150-9aaf04dc425c.png)

**Comment by thomasaugsten** [comment]
Welcher Zeitpunkt ist in der weißen Kachel als letzte Aktualisierung angegeben?

**Comment by jessinger179** [comment]
Welche Weiße Kachel ist gemeint ?

**Comment by jessinger179** [comment]
![1D0DD40F-A301-4587-9EB8-B95FA2CFE616](https://user-images.githubusercontent.com/100413215/155716207-765e3b28-6471-4b37-b7f1-fc5de715a915.png)

**Comment by thomasaugsten** [comment]
<img width="662" alt="image" src="https://user-images.githubusercontent.com/6367738/155716270-fe7a4376-4fcc-493a-92eb-dda16ed220ee.png">
Dieser Zeitpunkt

**Comment by jessinger179** [comment]
Dieser Zeitpunkt stimmt aber nicht, die App geht schon länger nicht mehr

**Comment by jessinger179** [comment]
![9178A307-5651-4F48-8A05-5C6E2DE5D6D1](https://user-images.githubusercontent.com/100413215/155716378-497a2698-a480-4b61-bf6a-d3e5184b21d1.png)

**Comment by thomasaugsten** [comment]
Kannst du bitte unter Fehlerberichte ein Fehlerbericht aktivieren und diesen dann heute Abend übermitteln und uns die Fehler ID mitteilen
<img width="400" alt="image" src="https://user-images.githubusercontent.com/6367738/155716874-fa0f2ca3-455c-47b1-896e-1beeef582676.png">

**Comment by Ein-Tim** [comment]
@jessinger179 

Könntest du bitte einen Screenshot von folgendem Bildschirm in den Systemeinstellungen hier schicken?  


`Einstellungen` ➡️ `Begegnungsmitteilungen` ➡️ `Status von Begegnungsaufzeichnungen` ➡️ `Begegnungsüberprüfungen`

Danke!

**Comment by jessinger179** [comment]
> @jessinger179
> 
> Könntest du bitte außerdem einen Screenshot von folgendem Bildschirm in den Systemeinstellungen hier schicken?
> 
> `Einstellungen` ➡️ `Begegnungsmitteilungen` ➡️ `Status von Begegnungsaufzeichnungen` ➡️ `Begegnungsüberprüfungen`
> 
> Danke!

![07FA7D30-48AC-45D9-A475-D59AF800C95D](https://user-images.githubusercontent.com/100413215/155717318-d9595274-6c58-488f-9544-9f3b0eb1b7c0.png)

**Comment by Ein-Tim** [comment]
Okay, Fehler 13 dürfte eigentlich nicht angezeigt werden. 

Bitte aktiviere den Fehlerbericht so wie von @thomasaugsten oben beschrieben, hoffentlich ist der Aufschlussreich.

**Comment by thomasaugsten** [comment]
Der Fehler wird immer noch angezeigt wenn die letzte Risikoberechnung älter als 48h ist.

**Comment by jessinger179** [comment]
Habe ich aktiviert und sende dann heute Abend die ID zu.

**Comment by Ein-Tim** [comment]
@thomasaugsten 

> Der Fehler wird immer noch angezeigt wenn die letzte Risikoberechnung älter als 48h ist.

Das ist das neue Behaviour mit 2.18, oder?

**Comment by larswmh** [comment]
Thanks for your report @jessinger179. We have created an internal ticket for it and will raise this topic internally. 
Internal Tracking ID: [EXPOSUREAPP-12056](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12056)

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
@jessinger179 

Tritt der Fehler immer noch auf?

**Comment by Ein-Tim** [comment]
@dsarkar I think this can be closed due to missing author feedback.

**Comment by dsarkar** [comment]
@jessinger179 Thanks for reporting. We are closing this issue now. Should you want to report another bug please open a new issue. Best wishes, DS

---
Corona-Warn-App Open Source Team
